Competition2017/src/competition/subsystems/feeder/SideFeederSubsystem.java

```diff
@@ -0,0 +1,186 @@
+package competition.subsystems.feeder;
+
+import org.apache.log4j.Logger;
+import com.ctre.CANTalon.FeedbackDevice;
+import com.ctre.CANTalon.TalonControlMode;
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SideFeederSubsystem extends BaseSubsystem {
+
+    private static Logger log = Logger.getLogger(SideFeederSubsystem.class);
+
+    protected final XCANTalon feederMotor;
+
+    public enum FeederSide {
+
+        Left, 
+
+        Right
+    }
+
+    private final FeederSide side;
+
+    protected final DoubleProperty intakePowerProperty;
+    protected final DoubleProperty ejectPowerProperty;
+    private final DoubleProperty feederTestingPowerStep;
+    private final DoubleProperty feederCurrentSpeed;
+    private final DoubleProperty feederTargetSpeed;
+    private final DoubleProperty feederOutputPower;
+    private final DoubleProperty feederTalonError;
+    
+    private final BooleanProperty atSpeedProp;
+    private final BooleanProperty enableFeederLogging;
```

Comment:
I think this code won't necessarily be permanent - but it would be nice to plot the response of the belt as balls are fed into it via the eggbeaters.